conf/application.conf

```diff
@@ -119,6 +124,11 @@ Test {
         start.path = "agent-services-account"
       }
 
+      government-gateway-authentication {
```

Comment:
Shouldn't it be different  for the different environment's  dev should be www-dev.tax.service.gov.uk etc.. And why not put in the appropriate app-configs ? and if they are the same you can put in app-config-base ;)